fastlane/Matchfile

```diff
@@ -7,6 +7,3 @@ storage_mode('google_cloud')
 google_cloud_bucket_name('a8c-fastlane-match')
 secrets_directory = File.join(Dir.home, '.configure', 'woocommerce-ios', 'secrets')
 google_cloud_keys_file(File.join(secrets_directory, 'google_cloud_keys.json'))
-
-# Use the decrypted API Key for authentication
-api_key_path(File.join(secrets_directory, 'app_store_connect_fastlane_api_key.json'))
```

Comment:
This had to be removed from `Matchfile` otherwise it would have always taken precedence.